### PR TITLE
fix(ci): reduce polling load, remove processed workload

### PR DESCRIPTION
**Pull request by milapjoshi**
Signed-off-by: milap joshi <milap.joshi@gmail.com>

## Summary

- remove processed workload from object to reduce load on API call
- reduce API polling by incresing polling interval
- return 'inconclusive' when test execution hit some failure 

## Test Plan

This change is already applied for HIL spirent for last 2/3 week
## Additional Information

- [ ] This change is backwards-breaking

### CODE REVIEW

**Review by tmdzk — APPROVED** [review]
Approved but I'd wait for @talkhasib  final approval before merging

**Review by talkhasib — APPROVED** [review]

**Comment by milapjoshi** [comment]
@quentinDERORY @tmdzk can you merge this.

**Pull request merged**
